model/ftn/wmesmfmd.ftn

```diff
@@ -1750,6 +1916,9 @@
 ! -------------------------------------------------------------------- /
 ! 1.  Check that required import fields show correct time stamp
 !
+      if (med_present) then
+      allUpdated = .true.
```

Comment:
This by-passes the import data-dependency. The import data-dependency restarts winds from an external component. During CMEPS restart runs this DataInitialize subroutine should check imported data time stamp to determine if CMEPS has restarted winds.

Reply:
Yes, we need to revisit this part once we have a configuration that restarts the model and uses CMEPS for coupling. I think there is no need to take action at this point.

Reply:
I think this could be a fairly major issue here.  If it's not addressed now, there needs to be an issue documenting that this is happening and that we need to look into it further.

Reply:
@JessicaMeixner-NOAA new issue - https://github.com/NOAA-EMC/WW3/issues/428 - is created to track it.